src/predict_triton.py

```diff
@@ -117,6 +118,28 @@ def infer():
   return {"predictions": return_payload}
 
 
+@app.route('/infer:raw', methods=["POST"])
```

Comment:
Is there a way to not have the FLASK server involved in this? Can the request go directly to the Triton server?

Reply:
If we want to maintain the same flow of "/infer" and "/infer:raw", then there's not a way without introducing an intermediate routing layer like nginx.

We still need something to align the urls.

We can directly expose the Triton server port, but then we have two separate api flows:

ex: 
Tokenized endpoint
http://10.24.0.1/infer 

Raw endpoint on Triton
http://10.24.0.1/v2/models/fastertransformer/infer

Reply:
The goal of the raw endpoint is performance, exposing the Triton server port is fine. Going behind FLASK or nginx could affect performance.

Reply:
Sounds good. Just tested it and all we need as an additional NodePort service exposed for the triton port. I'll abandon this PR and just roll it into the GKE changes.